walkthroughs/eks/o11y-cloudwatch.md

```diff
@@ -0,0 +1,99 @@
+# App Mesh with EKS—Observability: CloudWatch
+
+NOTE: Before you start with this part, make sure you've gone through the [base deployment](base.md) of App Mesh with EKS. In other words, the following assumes that an EKS cluster with App Mesh configured is available.
+
+
+First, create an IAM policy as shown below and attach it to the EC2 auto-scaling group of your EKS cluster, see also these [notes](https://eksworkshop.com/logging/prereqs/) for further details. Note that it doesn't matter to which node you attach the following policy as it will propagate automatically throughout the auto-scaling group:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "logs:DescribeLogGroups",
+                "logs:DescribeLogStreams",
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream",
+                "logs:PutLogEvents"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        }
+    ]
+}
+```
+
+Next, deploy Fluentd as a log forwarder using a `DaemonSet` as defined in the [fluentd.yml](fluentd.yml) manifest:
+
+```
+$ kubectl apply -f fluentd.yml
+
+# validate if the Fluentd daemon set is up and running:
+$ kubectl -n kube-system get po -l=k8s-app=fluentd-cloudwatch
+NAME                       READY   STATUS    RESTARTS   AGE
+fluentd-cloudwatch-7ls6g   1/1     Running   0          13m
+fluentd-cloudwatch-mdf9z   1/1     Running   0          13m
+```
+
+Now it's time to configure the virtual node `colorgateway-appmesh-demo` so it outputs its logs to `stdout`, which is in turn forwarded by Fluentd to CloudWatch. In order to configure the virtual node, use the console to set the log output on virtual node `colorgateway-appmesh-demo`.
+
+First, locate the virtual node `colorgateway-appmesh-demo` in the AppMesh console:
+
+![AppMesh console edit virtual node step 0](appmesh-log-0.png)
+
+Now, expand the 'Additional configuration' section and enter `/dev/stdout` in the 'HTTP access logs path' as shown in the following:
+
+![AppMesh console edit virtual node step 1](appmesh-log-1.png)
+
+Check by getting the virtual node configuration via the CLI:
+
+```
+$ aws appmesh describe-virtual-node \
```

Comment:
Output seems to be missing _Logging_ configuration.

Reply:
I double-checked: the logging part doesn't show up in the CLI output (and also not in the console). Removing this validation part for now until this is fixed.